src/run_context.rs

```diff
@@ -75,6 +75,24 @@ impl RunContext {
         Ok(dir.join(bundle_name).with_extension("mvb"))
     }
 
+    /// Path where script transactions are generated.
+    pub fn script_tx_output_path(&self, tx_name: &impl AsRef<Path>) -> Result<PathBuf, Error> {
+        let package_name = self.manifest()?.package.name.as_str();
+
+        // Create directory "<PACKAGE_PATH>/build/<PACKAGE_NAME>/script_transactions§/"
```

Comment:
```suggestion
        // Create directory "<PACKAGE_PATH>/build/<PACKAGE_NAME>/script_transactions/"
```

Reply:
Done

Reply:
good catch btw, I don't even know how to write that strange `§` letter :D